integrations/google_ai/src/haystack_integrations/components/generators/google_ai/gemini.py

```diff
@@ -62,14 +62,23 @@ class GoogleAIGeminiGenerator:
     ```
     """
 
+    def __new__(cls, *args, **kwargs):  # noqa: ARG003 - we cannote remove *args
```

Comment:
Can't we do this though? 🤔 
```suggestion
    def __new__(cls, *, **kwargs):
```